pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -593,7 +593,8 @@ func selectVictimsOnNode(
 		return nil
 	}
 	addPod := func(ap *v1.Pod) error {
-		nodeInfo.AddPod(ap)
+		podInfoToAdd := framework.NewPodInfo(ap)
```

Comment:
is this variable going to be used anywhere else? Otherwise all these lines could just be `nodeInfo.AddPod(framework.NewPodInfo(ap))`

Reply:
done. I will declare a new variable when it is used by other functions in later PR to make this one clear.